epochStart/metachain/trigger.go

```diff
@@ -22,6 +22,8 @@ import (
 
 var log = logger.GetOrCreate("epochStart/metachain")
 
+const minimumNonceToStartEpoch = 4
```

Comment:
Why 4?

Reply:
Because only from the 3-rd produced block we are starting to have cross notarized blocks. 4 just to be sure.